src/security/PeerConnector.cc

```diff
@@ -543,13 +559,35 @@ Security::PeerConnector::bail(ErrorState *error)
     Must(dialer);
     dialer->answer().error = error;
 
+    if (CachePeer *p = serverConnection()->getPeer())
```

Comment:
auto
const?

Reply:
c30b8b0a04
It is not possible to use const here.